frontend/src/routes/Embed.tsx

```diff
@@ -151,7 +169,15 @@ const Embed: React.FC<EmbedProps> = ({ query, queryRef }) => {
         </PlayerPlaceholder>;
     }
 
-    return <Player event={event} />;
+    const authorizedData = useAuthenticatedDataQuery(
+        event.id,
+        event.series?.id,
+        { authorizedData: event.authorizedData },
+    );
+
+    return authorizedData
+        ? <Player event={{ ...event, authorizedData }} />
+        : <PreviewPlaceholder embedded {...{ event }}/>;
 };
```

Comment:
I forgot to add the context.
That reminds me, since you also wanted to think about the context:
I tried moving that into a custom hook like the one in `PlayerContext.tsx`, but using that weirdly didn't have the same effect (read: it didn't have any effect) as just defining the state and context individually for each file it's needed in, like I do it now.

If, by any chance, you have some time and motivation to try this out, I'd be interested if you can get that to work. Of course, that only makes sense if we decide to stick with the context solution.